components/inputs/docs/input-search.md

```diff
@@ -82,11 +82,11 @@ When the input is cleared, the same event will be fired with an empty value.
 * `inline-help`: Help text that will appear below the input. Use this only when other helpful cues are not sufficient, such as a carefully-worded label.
 <!-- docs: end hidden content -->
 
-### Accessibility Properties
+## Accessbility
 
-To make your usage of `d2l-input-search` accessible, use the following property when applicable:
-
-| Attribute | Description |
-|---|---|
-| `description` | Use when label on input does not provide enough context. |
-| `label` | **REQUIRED** [Acts as a primary label on the input](https://www.w3.org/WAI/tutorials/forms/labels/). Not visible. |
+- While the component does not have a visible label, the use of a search icon helps make clear what its purpose is.
+	- While not required to meet WCAG, this [pattern](https://www.w3.org/WAI/WCAG2/supplemental/patterns/o1p07-icons-used/) is a great way to help individual with cognitive accessibility needs.
+	- This also allows the component to not have to worry about the contrast ratio of the placeholder text, as the contrast of the search icon meets that criteria.
+- It is important to note that `placeholder` is not a suitable replacement for using `label` or `description`, as once something is typed in, users will lose the context the placeholder provided.
+- Using a helper like [`announce`](https://github.com/BrightspaceUI/core/blob/main/helpers/announce.js) to inform screenreaders when search results have been returned helps vision-impaired users to inform them that they can start looking through the results.
```

Comment:
So using notifications to inform the user of form submissions is included in the W3C WAI documentation [here](https://www.w3.org/WAI/tutorials/forms/notifications/). But whether or not we want to do this as a whole at D2L isn't something that's been probably talked about, at least not in terms of it being considered a best practice. I'd initially had a preface in that line to inform people that it's not something that's mandatory, but could be a nice-to-have, but since some of the lines before it already cover things not being WCAG requirements, I didn't want it to be too repetitive.

Reply:
Ok, this might be a separate item to discuss with @geurts . I think there's both a design and technical consideration here. i.e. what is our best practice, and the if it's to announce something, how do we do it (`announce` vs visible search result in live region). I'm not sure that's been completely thought out.

Reply:
I think it's good to include guidance even though we don't have a firm pattern at this time. Could we just soften the language like "`Search results should be announced to screen reader users using a live region around the result summary or by using the announce helper`".

(Also, just note that we don't want periods at the end of bullets unless it contains more than one sentence)

Reply:
Yep, that sounds fine to me.

Reply:
Got it! I'll use that and also get rid of any of the periods at the end of the bullet points as well, I didn't realize that was best practice, I'll be sure to account for that more going forwards 😄